docs/packages/pkg/class_loader.rst

```diff
@@ -0,0 +1,24 @@
+.. spelling::
+
+    class_loader
+
+.. index:: ROS ; class_loader
+
+.. _pkg.class_loader:
+
+class_loader
+============
+
+-  `Official <https://official_class_loader>`__
+-  `Official <https://github.com/ros/class_loader>`__
+-  `Hunterized <https://github.com/hunter-packages/class_loader>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/class_loader/CMakeLists.txt>`__
+-  Added by `Krasimir Georgiev <https://github.com/kgeorgiev93>`__ (`pr-1899 <https://github.com/ruslo/hunter/pull/1899>`__)
+    - Contribution partially as part of work at `SeeByte Ltd. <http://www.seebyte.com>`__
+
+`class_loader` must be compiled with `BUILD_SHARED_LIBS=ON` for both libraries that are registered as plugins and libraries that load those plugins at runtime. If linking against `class_loader` statically, plugins would not be loaded correctly at runtime, because both the plugin and the library loading it would use their own copy of `class_loader`.
```

Comment:
Since you set `BUILD_SHARED_LIBS=ON` by default the right note is:
```
By default BUILD_SHARED_LIBS=ON used for class_loader because ...
```

Reply:
```
`class_loader`
```
Not correct for RST, should be
```
``class_loader``
```

Instructions to build documentation:
- https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#add-documentation